src/snowflake/snowpark/_internal/valid_view_merge_functions.py

```diff
@@ -0,0 +1,191 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+# aggregation, data generation or nondeterministic with unique reference
+VALID_VIEW_MERGE_FUNCTIONS = (
```

Comment:
if it's complement, we can programmatically generate this list?

Reply:
Oh, @sfc-gh-aalam , @sfc-gh-jdu  sorry, that is not used anywhere, i forgot to remove that file, it is the same as VALID_PROJECTION_MERGE_FUNCTIONS.

@sfc-gh-jdu I have thought about this, they way i try to get the set of functions is to run a sql query to get all functions and filtered based on some properties. I also did some manual verification to make sure the current set is valid. This is a very restricted set at this moment.

Since this requires sync with server side, so i didn't really do it on session init to avoid unnecessary overhead. Let me put some instructions here about  how to pull the functions and property for check.

Reply:
I see, thanks